assets/src/features/leaving-page/components/Questionnaire.tsx

```diff
@@ -44,7 +44,7 @@ const Questionnaire: FC<QuestionnaireProps> = ({ onSubmitClick }) => {
 
   const ratingNames = ["video", "audio", "screensharing"] as const;
   const isOneQualityRated = ratingNames.map(watch).some(isNotNil);
-  const canSubmit = emailFilled && isOneQualityRated;
+  const canSubmit = emailFilled || isOneQualityRated;
```

Comment:
At first glance, this doesn't look like a valid change. IIUC, now you can submit the form with email without rating or with a rate without email

Reply:
That's a valid point. I'll change it so the button is disabled if and only if none of the ratings are clicked. Nonetheless, the button does not need to be disabled when the email is invalid or wrongly filled since the `react-hook-form` will prevent it from being sent.